Automattic/pocket-casts-ios
# [Deselecting Chapters] Add feature flag

| 📘 Part of: #1396 |
|:---:|

Adds the remote feature flag for the upcoming deselecting chapters feature.

## To test

1. Go to `AppDelegate` and remove the `#if !DEBUG` inside `updateRemoteFeatureFlags`
2. On line `291` change `2.hour` to `10.seconds`
3. Run the app
4. ✅ Go to Profile > Settings > Beta Features > `deselectChapters` should be toggled off
5. Go to Firebase Console and change `deselect_chapters` to `true`
6. Re-run the app
7. ✅ Go to Profile > Settings > Beta Features > `deselectChapters` should be toggled on
8. Go to Firebase Console and change `deselect_chapters` to `Use in-app default`
6. Re-run the app
7. ✅ Go to Profile > Settings > Beta Features > `deselectChapters` should be toggled off

## Checklist
